src/thunderbots/software/grsim_communication/main.cpp

```diff
@@ -79,8 +83,10 @@ int main(int argc, char** argv)
         // The callbacks will populate the primitives vector
         ros::spinOnce();
 
-        grsim_backend.updateBackendTeam(friendly_team);
-        grsim_backend.sendPrimitives(primitives);
+        primitives.emplace_back(std::unique_ptr<Primitive>(
+            new MovePrimitive(0, Point(0, 0), Angle::zero(), 0.0)));
```

Comment:
Can we remove this (for testing, not for PR).

Reply:
fixed